_includes/highlights.html

```diff
@@ -32,7 +32,7 @@ <h2 class="usa-graphic-list__heading">Building bridges</h2>
                 <span class="graphic-icon__container">
                   {% asset homecoming-cropped.jpg class="usa-media-block__img img-circle" alt="PIF Fellow speaking" %}
                   <div class="graphic-icon">
-                    {% asset icons/people.png alt="people icon" %}
+                    {% asset icons/people.png alt="" %}
```

Comment:
Leaving the alt tag blank means that a screen reader won't read out `people icon` when it hits this. It will act like there's nothing there.